test/assertion/get_tgt_val/test_assert_get_tgt_val.py

```diff
@@ -0,0 +1,163 @@
+import unittest
+
+from quantestpy import PauliCircuit, assert_get_tgt_val
+
+
+class TestAssertGetTgtVal(unittest.TestCase):
+    """
+    How to execute this test:
+    $ pwd
+    {Your directory where you git-cloned quantestpy}/quantestpy
+    $ python -m unittest test.assertion.get_tgt_val.test_assert_get_tgt_val
+    .
+    ----------------------------------------------------------------------
+    Ran 1 tests in 0.044s
+
+    OK
+    $
+    """
+
+    def setUp(self) -> None:
+        """
+        This is the circuit in Figure 7 in arxiv:1805.03662
+        """
+        self.pc = PauliCircuit(1+4+11+4)
```

Comment:
It would be nicer to use the circuit in Figure 10 in arxiv:1805.03662 or Figure 1 in arxiv:1812.00954.